src/templates/ObservabilityPackDetails.js

```diff
@@ -226,7 +226,15 @@ const ObservabilityPackDetails = ({ data, location }) => {
             <Tabs.BarItem id="alerts" count={pack.alerts?.length ?? 0}>
               Alerts
             </Tabs.BarItem>
-            <Tabs.BarItem id="data-sources">Data sources</Tabs.BarItem>
+            <Tabs.BarItem
+              id="data-sources"
+              count={
+                (pack.instrumentation?.length ?? 0) +
+                (pack.documentation ? 1 : 0)
```

Comment:
Sorry to toss another one on here! But the documentation field will also be an array

Reply:
thank you 😂 I had it in my head that was a string